Security and Site Policy validations

srinutamada opened the pull request:
Hi Bert,

We have added security and site policy validator to the PnP site core solution.

Thanks and Regards,
Srinu Tamada

msftclas commented:
Hi __@srinutamada__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.microsoft.com.
    </span>

TTYL, MSBOT;

srinutamada commented:
Hi Bert,

We have implemented pages, files and workflow validators to the PnP site core and added to the same pull request. Could you please review and branch?

Regards,
Srinu Tamada
